Bottom: 9d5e87ec3982847e89b8102eccac7589d928ca56
Top:    1bc147946265bc52601a3370487534760c5a11ba
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:37:47 +0000

TMP alterartions to build-all.sh


---

diff --git a/w32/build-all.sh b/w32/build-all.sh
index 04aec82..689d307 100755
--- a/w32/build-all.sh
+++ b/w32/build-all.sh
@@ -2,9 +2,6 @@ NEVERBUILD="
 "
 
 NOBUILD="
-"
-
-BUILD="
 jpeg
 zlib
 libpng
@@ -17,6 +14,9 @@ glib
 atk
 cairo
 pango
+"
+
+BUILD="
 gtk+
 gd
 pcb
